lib/vssh/libssh2.c

```diff
@@ -568,6 +568,10 @@ static CURLcode ssh_knownhost(struct connectdata *conn)
       /* DEFER means bail out but keep the SSH_HOSTKEY state */
       result = sshc->actualcode = CURLE_PEER_FAILED_VERIFICATION;
       break;
+    case CURLKHSTAT_FINE_REPLACE:
+      /* remove old host+key that doesn't match */
+      if(host)
+        libssh2_knownhost_del(sshc->kh, host);
```

Comment:
I presume this is an intentional fall-through here, and then you must add a `/* FALLTHROUGH */` comment. Preferably on its own line, below. Some compilers will complain otherwise.

Reply:
done 👍